src/output-json.h

```diff
@@ -50,6 +50,7 @@ typedef struct JsonAddrInfo_ {
     Port sp;
     Port dp;
     char proto[JSON_PROTO_LEN];
+    bool log_port;
```

Comment:
can you add a comment here to doc the new field?

Reply:
Why not remove it as proposed in the previous version of this PR ?

Reply:
I can remove it. @victorjulien what would you prefer?

Reply:
I think I missed the prev discussion. What was the suggestion?

Reply:
The suggestion was to use a variable instead of adding a new field inside the struct.
(https://github.com/OISF/suricata/pull/10985#discussion_r1584546716)

Reply:
Think I have no strong preference.